src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -473,14 +471,14 @@ protected function _bundleTranslatedFields($entity)
         }
 
         $results = $this->_findExistingTranslations($find);
-        $model = $this->config('conditions.model');
+        $alias = $this->_table->alias();
 
```

Comment:
I've propsed `Table::name()` in #5984 which would, I think, remove the need for the model key.